docs/openapi.yaml

```diff
@@ -4,9 +4,21 @@ info:
   title: "IUDX File Server APIs"
   version: '1.0'
   description: |
-    The The File Server is IUDX File store which allows upload, download and discovery of Files.
-    For Upload it allows uploading Files with group id or resourcer item id.
-    For Discovery it allows temporal queries to search files data within time ranges.
+  
+    # Introduction
+    The File Server is IUDX File store which allows upload, download and discovery of Files.
+    For Upload it allows uploading Files with group id or resource item id, it also allow user to attach additional metadata fields to the uploaded file.
+    For Discovery it allows temporal queries to search files data within time ranges, geo parameters. It also allows user to list file metadata using resource group id or resourfce item id. 
+    
+    # API HTTP Responses
+    Apart from the response codes specified in each API, the API server may respond with certain 4xx and 5xx error codes which are related to common API Gateway behaviours. The application should address them accordingly.
+    
+    # Pagination
+    File server also offers way to paginate the result for queries. If a query returns large number of records then user can use additional parameters in query parameters to limit numbers of records to be returned.
+    

```

Comment:
Lets mention that the pagination is applicable only for /entities API

Reply:
done